### ISSUE TITLE
Create a "similar bug" recommender

### ISSUE THREAD

**Issue by marco-c** [marco-c/bugbug#26]
"similar" not in terms of duplicate bug, but in terms of features. E.g. two bugs that are completely unrelated from each other, but that are of the same type and have similarly sized patches.

**Comment by spectre1811** [comment]
It means we have to build a kind of recommender system based upon the features and the size of the patches extracted from the bug info.?

**Comment by marco-c** [comment]
> It means we have to build a kind of recommender system based upon the features and the size of the patches extracted from the bug info.?

Yes (note that the size of the patches is just one of the many features that we can consider).

**Comment by spectre1811** [comment]
Yeah i figured that thing. So can i work upon this thing as my gsoc project because i think this would take some time to be completed

**Comment by marco-c** [comment]
Yes, this is a big enough project for a GSoC.

**Comment by spectre1811** [comment]
I want to work on this. Can i be assigned this?

**Comment by marco-c** [comment]
@spectre1811 see the contributing rules in README, an issue is assigned only when there is a PR to fix it.

**Comment by marco-c** [comment]
See also #517.
